src/routes/index.tsx

```diff
@@ -5,10 +5,18 @@ import DailyDropffsReport from "../components/daily-dropoffs-report";
 import ProofingOverTimeReport from "../components/proofing-over-time-report";
 import { Router } from "../router";
 import HomeRoute from "./home-route";
-import createReportRoute from "./report-route";
+import createReportRoute, { ReportRoute } from "./report-route";
 import { Scale } from "../contexts/report-filter-context";
+import ALL_ROUTES from "./all";
 
-export const ROUTES = {
+/**
+ * Requires that all keys in ALL_ROUTES have a matching key in this object
+ */
+type ReportRoutes = {
+  [Property in keyof typeof ALL_ROUTES]: ReportRoute;
+}
+
+const reportRoutes: ReportRoutes = {
```

Comment:
I confirmed that this will yell if `reportRoutes` is missing a key that's in `ALL_ROUTES`